pkg/syncer/cnsoperator/controller/cnsregistervolume/cnsregistervolume_controller.go

```diff
@@ -201,11 +201,15 @@ func (r *ReconcileCnsRegisterVolume) Reconcile(request reconcile.Request) (recon
 	log.Debugf("CNS Volume create spec is: %+v", createSpec)
 	vol, err := r.volumeManager.CreateVolume(ctx, createSpec)
 	if err != nil {
-		volumeID = instance.Spec.VolumeID
-	} else {
-		volumeID = vol.Id
-		log.Infof("Created CNS volume with volumeID: %s", volumeID)
+		msg := fmt.Sprintf("Failed to create CNS volume with error: %+v", err)
```

Comment:
Failed ==> failed
Error strings should not be capitalized, since they are usually printed following other context

Reply:
Done.